charts/agent-operator/templates/operator-deployment.yaml

```diff
@@ -39,6 +38,18 @@ spec:
         {{- if and .Values.kubeletService.namespace .Values.kubeletService.serviceName }}
```

Comment:
would something like this maybe simplify the template here a bit? i found it a bit difficult to follow at first glance:

```yaml
        {{- if or .Values.kubeletService .Values.extraArgs }}
        args:
          {{- if .Values.kubeletService }}
          - --kubelet-service={{ .Values.kubeletService.namespace }}/{{ .Values.kubeletService.serviceName }}
          {{- end }}
          {{-  if .Values.extraArgs  }}
          {{- range .Values.extraArgs  }}
          - {{ . }}
          {{- end }}
          {{- end }}
        {{- end }}
```

Reply:
Totally! I've done a bad job at that 😞  Have refactored with your approach - thanks for the hint 👍